src/libPMacc/include/math/vector/Vector.hpp

```diff
@@ -660,7 +662,7 @@ typename T_Navigator,
 template <typename, int> class T_Storage
 >
 HDINLINE Vector<T_Type, T_Dim>
-operator*(const T_Type& lhs, const Vector<T_Type, T_Dim, T_Accessor, T_Navigator, T_Storage>& rhs)
+operator*(typename boost::call_traits<T_Type>::param_type lhs, const Vector<T_Type, T_Dim, T_Accessor, T_Navigator, T_Storage>& rhs)
```

Comment:
some here `typename Vector<T_Type, T_Dim, T_Accessor, T_Navigator, T_Storage>::type` for `T_Type`